index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="main.js" defer></script>
+    <title>Document</title>
+</head>
+<body>
+    <header >
+        <nav >
+            <ul class="tallbar">
+                <li><a href="#"><img src="img/Rectangle.png" alt=""></a></li>
+                <li><a href="#"><img src="img/twitter.png" alt=""></a></li>
+                <li><a href="#" class="English">English</a></li>
+                <li><a href="#" class="My-Page">My Page</a></li>
+                <li><a href="#" class="Logout">Logout</a></li>
+            </ul>
+        </nav>
+
+        <div class="hearder">
+            <div><img src="img/mobile_menu 1.png" alt="" class="logo" onclick="showMenu()"></div>
```

Comment:
- [ ] Well done on your progress so far! I've noticed that the `alt` attribute is missing for several images throughout your code. It's important to provide descriptive and meaningful `alt` attributes for images to enhance accessibility and ensure a better user experience, especially for users who may rely on screen readers. Please take the time to review each image and add appropriate `alt` attributes to them. This attention to detail will make your project more inclusive and user-friendly. Keep up the great work! 🌟📝🚀